lore/learningobjects/models.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+
+
+class Course(models.Model):
```

Comment:
I think it would be nice to create a base model as per a suggestion from @ShawnMilo that these all inherit from that add auditing fields, date_modified, date_created